vnext/PropertySheets/React.Cpp.props

```diff
@@ -67,6 +67,10 @@
     <Import Project="$(ConfigurationType)\$(Configuration).props" />
   </ImportGroup>
 
+  <PropertyGroup>
+    <CppStandard Condition="'$(CppStandard)'==''">stdcpp17</CppStandard>
```

Comment:
Shall we switch the build to cpp20?

Reply:
not yet but once C++20 support is complete, we can consider it. Office and others may stay on 17 for longer.

Reply:
It shouldn't matter for Office since we're consuming from a binary instead of building from source.

Reply:
there may be STL ABI breaks between 17 and 20 though, wouldn't want to risk that :)

Reply:
Plus, there is some code that doesn't yet build with c++20 - some in folly, and some in RNW